docs/manage-statistics-in-gravitino.md

```diff
@@ -0,0 +1,215 @@
+---
+title: "Manage statistics in Gravitino"
+slug: /manage-statistics-in-gravitino
+date: 2025-08-21
+keyword: statistics management, statistics, statistic, Gravitino
+license: This software is licensed under the Apache License version 2.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Introduction
+
+Starting from 1.0.0, Gravitino introduces statistics of tables and partitions.
+
+This document provides a brief introduction by using both Gravitino Java client and
+REST APIs. If you want to know more about the statistics system in Gravitino, please refer to the
+Javadoc and REST API documentation.
+
+Statistics only support the custom statistics, which names must start with `custom-`.
+Gravitino will support build-in statistics in the future.
+
+
+## Table Statistic operations
+
+### Update table statistics
+
+You can update the statistics of a table by providing the statistics key and value.
+
+The request path for REST API is `/api/metalakes/{metalake}/objects/{metadataObjectType}/{metadataObjectName}/statistics`.
```

Comment:
for table statistics, the `{metadataObjectType}` is a fixed value `table`, not a variable